### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by johnathanbostrom**

### CODE REVIEW

File: server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+############################################################
+###Step 1
+############################################################
+FROM fpco/stack-build:lts-11.0 as build
+# FROM haskell:latest as build
```
Comment (tlentz) [inline server/Dockerfile]: is this supposed to be commented out?

File: server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+RUN stack setup
+RUN stack install hpack
+RUN stack install intero
```
Comment (tlentz) [inline server/Dockerfile]: intero is only used for vscode when you are developing and needs the .stack-work folder in the parent directory that you have open in vscode.  does this work in the docker container?

**Review by tlentz — APPROVED** [review]
Just a few questions, other than that LGTM

File: server/Dockerfile (hunk 2 above)
Comment (johnathanbostrom) [reply server/Dockerfile]: The intention is for this to eventually let you use vscode without stack or intero installed on your machine.  It needs testing though.  I could remove intero from the image until I am ready to test it out.

**Pull request merged**